clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/caching/agent/KubernetesCachingAgent.java

```diff
@@ -106,7 +106,12 @@ protected KubernetesCachingAgent(
   @Nonnull
   private ImmutableList<KubernetesManifest> loadNamespaceScopedResources(
       @Nonnull Iterable<KubernetesKind> kubernetesKinds) {
-    return getNamespaces().parallelStream()
+    return getNamespaces()
```

Comment:
This feels like it belongs in a separate commit, or maybe even PR since it's not specific to the sql scheduler.

Reply:
Yeah, I'll have a separate PR for this one.